lang/en/theme_boost_union.php

```diff
@@ -242,6 +242,15 @@
 $string['courseheaderimagelayoutheadingabove'] = 'Course title above of course image';
 $string['courseheaderimageposition'] = 'Course header image position';
 $string['courseheaderimageposition_desc'] = 'With this setting, you control the positioning of the course header image within the course header image container. The first value is the horizontal position, the second value is the vertical position.';
+// ... Section: Course index.
+$string['courseindexheading'] = 'Course Index';
+// ... ... Setting: Course index.
+$string['courseindexmodiconenabled'] = 'Display course module icons in course index';
+$string['courseindexmodiconenabled_desc'] = 'When enabled, the corresponding course module icon is displayed in front of the index row. In doing so, the course module icon is replacing the course completion indicator which is moved from the front to the right side of the index row.';
```

Comment:
`$string['courseindexmodiconenabled_desc'] = 'When enabled, the corresponding course module icon is displayed in front of the index row. In doing so, the course module icon is replacing the course module completion indicator which is moved from the front to the right side of the index row.'`

Better not to confuse course completion indicator and course module completion indicator.

Reply:
It seem to me that watching https://www.youtube.com/watch?v=PHvklKLq2T4 I think it's best to talk only of 'activities'.
The double mentioning of 'activities and resources' is to be dropped and 'resources' is just being seen as a 'category' of an activity, such as 'assessment', 'collaboration', 'communication' et cetera.

So, me, personally would just call it 'activity' what you see as single course module, e.g. 'quiz 1', 'assessment abc', 'workshop α' and 'activity type' when referencing to what influences the icon, e.g. quiz, assessment, workshop.

In that spirit:
`
$string['courseindexacttypeiconenabled'] = 'Display activity type icons in course index';
$string['courseindexacttypeiconenabled_desc'] = 'When enabled, the corresponding activity type icon is displayed in front of the index row. In doing so, the course activity type is replacing the course completion indicator which is moved from the front to the right side of the index row.';
`